hr_holidays_public/wizards/holidays_public_next_year_wizard.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Trobz
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="holidays_public_next_year_wizard_view" model="ir.ui.view">
```

Comment:
Could you indent the records more to the right, making this xml consistent with the other views?